README.md

```diff
@@ -41,6 +42,22 @@ Registration is the task of aligning or developing correspondences between
 data. For example, in the medical environment, a CT scan may be aligned with a
 MRI scan in order to combine the information contained in both.
 
+[//]: # (numfocus-fiscal-sponsor-attribution)
+
+The ITK project is governed by the [Insight Software Consortium bylaws](./GOVERNANCE.md)
```

Comment:
As in the Shogun and EconArk examples:

https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md

I think *an open governance model* vs *by the Insight Software Consortium bylaws* more directly describes the governance model .

Reply:
Changed in 5ab5522.